<launch>
<include file="$(find carla_ros_bridge)/client.launch"  pass_all_args="True"/>
<param name ="/use_sim_time" value="true"/>
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find carla_ros_bridge)/config/carla_default_rviz.cfg.rviz"/>

 <!-- Axes -->
  <param name="axis_linear" value="2" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="13" type="double"/>
  <param name="scale_angular" value="10" type="double"/>

<node pkg="carla_ros_bridge" name="calcdis_node" type="calc_drive_distance.py" output="screen"/>
</launch>
